source/agora/consensus/Fee.d

```diff
@@ -515,3 +515,61 @@ public class FeeManager
         assert(tot_fee == Amount(0));
     }
 }
+
+unittest
+{
+    import agora.consensus.state.UTXOSet;
+    import agora.common.Hash;
+
+    const db_file = "fee_man.tmp";
+    auto fee_man = new FeeManager(db_file, new immutable(ConsensusParams));
```

Comment:
You can just introduce a new `private` constructor which takes a database

Reply:
Yeah, will do that